contents/javascript/Javascript-framework-Angular.md

```diff
@@ -63,8 +61,15 @@ export class HeroesComponent {
 
 The `@Component` tag is used to mark a component. It provides the metadata that determines how the component is processed during runtime.
 
-* The use of `selector: app-heroes` is how Angular identifies this component in a template and renders it.  
-* The `hero.name` used in the template, refers to the name property defined in the above component.
+* The use of `selector: app-person` is how Angular identifies this component in a template. The following code constitutes the template of `app.component.html`.
```

Comment:
>The following code constitutes the template of `app.component.html`

It is odd that this component simply includes the other component and has nothing else. Maybe you  can make it like this? Does that make senes?
```html
<h1>Person Data</h1>

<app-person></app-person>

<hr/>
```

Reply:
Yup you are right, makes sense to add a header at least